KaitaiStream.cs

```diff
@@ -280,12 +287,18 @@ public double ReadF8le()
 
         #region Unaligned bit values
 
+        /// <summary>
+        /// ???
+        /// </summary>
         public void AlignToByte()
         {
             Bits = 0;
             BitsLeft = 0;
         }
```

Comment:
"Clears the temporary buffer which holds not-yet-consumed parts of the byte, which might have left over after last unaligned bit read operation. Effectively, aligns the pointer in the stream to next whole byte, discarding the rest of partially byte, if it existed. Does nothing if unaligned bit reading is not used."

Reply:
Fixed.